website/docs/usage/projects.md

```diff
@@ -43,11 +43,12 @@ and experiments, iterate on demos and prototypes and ship your models into
 production.
 
 <Grid narrow cols={3}>
+<Integration title="Comet" logo="comet" url="#comet">Manage your experiments and models</Integration>
 <Integration title="DVC" logo="dvc" url="#dvc">Manage and version your data</Integration>
-<Integration title="Prodigy" logo="prodigy" url="#prodigy">Create labelled training data</Integration>
-<Integration title="Streamlit" logo="streamlit" url="#streamlit">Visualize and demo your pipelines</Integration>
 <Integration title="FastAPI" logo="fastapi" url="#fastapi">Serve your models and host APIs</Integration>
+<Integration title="Prodigy" logo="prodigy" url="#prodigy">Create labelled training data</Integration>
 <Integration title="Ray" logo="ray" url="#ray">Distributed and parallel training</Integration>
+<Integration title="Streamlit" logo="streamlit" url="#streamlit">Visualize and demo your pipelines</Integration>
 <Integration title="Weights &amp; Biases" logo="wandb" url="#wandb">Track your experiments and results</Integration>
 </Grid>
```

Comment:
There isn't a Comet project, so the entry here should be removed. Let's just keep the original ordering as well.

Reply:
Ok, I'll remove the wandb Project as well.

Reply:
No - please don't - cf my comment above.

Reply:
Fixed!